app/Http/Controllers/AdminController.php

```diff
@@ -34,6 +34,17 @@ public function showUsers()
             return view('admin.users.allusers')->with('all_users', $all_users);
         }
     }
+    public function deactivate($id)
```

Comment:
you can change your `deactivate($id)` function into this `deactivate(User $user)` and below there's no need for `User::destroy($id)`, you can just use `$user->delete()`. make sure in your web routes that's pointing to the deactivate function to change `{id}` to `{user}`

Reply:
I'll fix it.